app/admin/edit-user/edit-user.component.html

```diff
@@ -0,0 +1,56 @@
+<md-card class="default-card">
+	<span *ngIf="user.id === -1">{{ 'loading' | translate }}...</span>
+	<md-card-content *ngIf="user.id !== -1">
+		<h2>{{ 'edit user' | translate }} {{ user.name }}</h2>
+		<form>
+			<md-grid-list cols="2" rowHeight="4em">
+
+				<md-grid-tile>
+					<md-input [(ngModel)]="user.lastName" required type="text" id="lastName" placeholder="{{'last name' | translate}}" name="name"
+						#lastName="ngModel">
+						<md-hint [hidden]="lastName.valid" class="error-hint">
+							<span>{{ 'name required' | translate }}</span>
+						</md-hint>
+					</md-input>
+				</md-grid-tile>
+
+        <md-grid-tile rowspan="4">
+          <!-- temporary until upload-picture component can be used -->
+					<div style="height: 100%;">
+            <md-card class="Image" style="height: 80%;"></md-card>
+            <button md-raised-button color="primary" class="uploadButton" style="height: 20%;">{{'upload picture' | translate}}</button>
+          </div>
```

Comment:
Not sure if we can reuse it, because it will use another API call than the admin module should do.